scheduler.js

```diff
@@ -0,0 +1,16 @@
+const talenta = require("./index");
+const { cookiesTalenta, longitude, latitude, timeClockIn, timeClockOut } = require("./config");
+
+(async () => {
+  if (timeClockIn && timeClockOut) {
```

Comment:
what do you think? could you just check both or one at a time like
```
if (timeClockIn) {
  //scheduler clockIn
}

if (timeClockOut) {
  //scheduler clockOut
}

Reply:
I think schedulers only for both, so if condition not meet, you could throw Error with relevant message and stop the application

Reply:
> what do you think? could you just check both or one at a time like
> 
> ```
> if (timeClockIn) {
>   //scheduler clockIn
> }
> 
> if (timeClockOut) {
>   //scheduler clockOut
> }
> ```

back to your argument in here, 
https://github.com/ans-4175/talenta-api/pull/9#discussion_r722308448

so that's why I decided to do both scheduler for clockin and lockout in one place. 
Also it is to prevent user, from forgetting to lockout if they just run scheduled for clockin. 
So, both must be run at the same time.

Reply:
> I think schedulers only for both, so if condition not meet, you could throw Error with relevant message and stop the application

alright

Reply:
because the scheduler is running inside pm2, do you have any idea how to stop/cancel the process if there is an error, or condition not meet?
if I just throw an Error, the pm2 will still running
<img width="1440" alt="Screen Shot 2021-10-05 at 23 41 03" src="https://user-images.githubusercontent.com/39824404/136066080-aedd15f9-736f-4442-9d55-0993d4a7efc8.png">
.

Reply:
`process.exit(1)`
it will signaling exit system with error message